project-koku/koku
# Adding small tweak to initial summary months

Fixes [COST-2683](https://issues.redhat.com/browse/COST-2683)

Changes proposed in this PR:
* Adding a cap on summary period based from initial ingest num months


Testing Instructions:
1. Add OCI source with data outside the initial ingest num months.
2. See that we download but only summarise data within the num months period

---
Additional Context:
